cpp/cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -2449,8 +2445,6 @@ macro(build_grpc)
       -DgRPC_PROTOBUF_PROVIDER=package
       -DgRPC_SSL_PROVIDER=package
       -DgRPC_ZLIB_PROVIDER=package
-      -DCMAKE_CXX_FLAGS=${GRPC_CMAKE_CXX_FLAGS}
```

Comment:
oops, should not remove this!

Reply:
We will be able to remove this in #8881 . :-)
#8881 uses gRPC 1.33.2.